system-addon/content-src/components/CollapsibleSection/_CollapsibleSection.scss

```diff
@@ -3,15 +3,15 @@
 
     .click-target {
       cursor: pointer;
+      vertical-align: top;
     }
 
-    .icon {
-      &.icon-arrowhead-down {
-        margin: -3px 0 0 8px;
-      }
+    .icon-arrowhead-down,
+    .icon-arrowhead-forward {
+      margin: -1px 0 0 8px;
 
-      &.icon-arrowhead-forward {
-        margin: -2px 0 0 8px;
+      &:dir(rtl) {
+        margin: -1px 8px 0 0;
```

Comment:
Probably just do `margin-top: -1px` and `margin-inline-start: 8px` to not need the separate rtl check.